moped-database/metadata/tables.yaml

```diff
@@ -1,6 +1,16 @@
 - table:
     name: component_arcgis_online_view
     schema: public
+  array_relationships:
+    - name: componentToExplodedGeometryComponent
+      using:
+        manual_configuration:
+          column_mapping:
+            project_component_id: project_component_id
+          insertion_order: null
+          remote_table:
+            name: exploded_component_arcgis_online_view
+            schema: public
```

Comment:
This is a real relationship, and it's usable when you're looking at a handful of AGOL records, but it's not performant enough to use when asking for the full table. 

Because of this, I am not using the relationship in the program, but rather, just executing two queries. 

But since it's a genuine and real relationship, I'm thinking we should keep it. But I would be happy to zap it out of there too.

Reply:
I'm not sure that I fully understand, but if this isn't performant enough to be used in the front end (its not, right?) then why include it?

Reply:
You bet, thanks for this! de42ca4273b442f7c5e96817f77fd0bfea33b7f0